### PR TITLE
update for sound

### PR DESCRIPTION
**Pull request by howard-stearns**
Update crowd scripts to use avatar sound.

Currently just piano, which will be played four of the 25 avatars surrounding you, in rotation. 

(Subject to however many in instances of crowd-agent.js are running on the domain. Also fixes a bug in reporting that.)

### CODE REVIEW

**Comment by howard-stearns** [comment]
Buzzes when you stop the script. I don't know if that's a bug in the script, the agent, the mixer, or the client.

Note that although this uses "agent mic sound", it does not use compression yet.

**Comment by hifi-gustavo** [comment]
Build console output for commit: c5f889a34dae02670edee1d972a178c26ecfe522 available at: https://gist.github.com/52bad2bf4bf5be625394390482c7038b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8759-c5f889a.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8759-c5f889a.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7c5dd89372a0f420c8f46a26902ecc63400b70e6 available at: https://gist.github.com/52bad2bf4bf5be625394390482c7038b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8759-7c5dd89.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8759-7c5dd89.dmg

File: scripts/developer/tests/performance/summon.js [hunk 1]
```diff
@@ -93,11 +118,12 @@ Script.scriptEnding.connect(function () {
 
 messageSend({key: 'HELO'}); // Ask agents to report in now.
 Script.setTimeout(function () {
+    var total = AvatarList.getAvatarIdentifiers().length;
     if (0 === summonedAgents.length) {
         Window.alert("No agents reported.\n\Please run " + MINIMUM_AVATARS + " instances of\n\
 http://cdn.highfidelity.com/davidkelly/production/scripts/tests/performance/crowd-agent.js\n\
```
Comment (sethalves) [inline scripts/developer/tests/performance/summon.js]: is this url still right?

Comment (howard-stearns) [inline scripts/developer/tests/performance/summon.js]: good point. @davidkelly, could you update that url with the contents from this PR?

Comment (davidkelly) [inline scripts/developer/tests/performance/summon.js]: That url is the version 1 of crowd-agent.  I'm guessing we can (for now) update that in S3 to this version of crowd-agent.js.  If that sound ok (just for now), I'm happy to update.

**Pull request merged**